tests/unit/assessment/test_pipelines.py

```diff
@@ -21,23 +22,57 @@ def test_pipeline_assessment_with_config(mocker):
         )
     ]
 
-    ws = Mock()
+    ws = workspace_client_mock(clusters="job-source-cluster.json")
     config_dict = {
         "spark.hadoop.fs.azure.account.auth.type.abcde.dfs.core.windows.net": "SAS",
         "spark.hadoop.fs.azure.sas.token.provider.type.abcde.dfs."
         "core.windows.net": "org.apache.hadoop.fs.azurebfs.sas.FixedSASTokenProvider",
         "spark.hadoop.fs.azure.sas.fixed.token.abcde.dfs.core.windows.net": "{{secrets/abcde_access/sasFixedToken}}",
     }
     ws.pipelines.get().spec.configuration = config_dict
+    ws.pipelines.get().spec.clusters = mock_pipeline_cluster
+    ws.cluster_policies.get(policy_id="single-user-with-spn").definition = (
```

Comment:
Look at "__init__.py" in the same folder and do it like there. Modify that file if needed

Reply:
Modified the code accordingly